Specification/5._Application_level_conventions.md

```diff
@@ -8,33 +8,31 @@ permalink: /Specification/5._Application_level_conventions/
 
 ### Message type ID
 
-As specified in the CAN bus transport layer specification, the valid range for message type ID is from 0 to 65535,
-inclusive.
+The valid range for *message type ID* is from 0 to 65535, inclusive (specified in [CAN bus transport layer](/Specification/4._CAN_bus_transport_layer)).
```

Comment:
FYI, "As specified in the CAN bus transport layer specification" is a bit hard to parse, and isn't the most important information. I pushed that to end of sentence and made a link. This would read even better as "see Can bus ..." - but perhaps it is useful to make it clear that this is part of the specification.

Reply:
Seems like the word `specification` is missing here. Perhaps it should be `specified in [CAN bus transport layer specification](/Specification/4._CAN_bus_transport_layer)`?

Reply:
The problem here is that "specified in the XXXX specification" sounds redundant - but what I had is incorrect, because we do specify in a specification, not in a layer. 

I will change this to "see [CAN bus transport layer specification]" - IMO this shows where the value is specified/set. OK?